aleph/migrate/versions/d46fc882ec6b_api_key_expiration.py

```diff
@@ -0,0 +1,30 @@
+"""API key expiration
+
+Revision ID: d46fc882ec6b
+Revises: c52a1f469ac7
+Create Date: 2024-05-02 11:43:50.993948
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
```

Comment:
What are revision identifiers used for, something to do with upgrade tasks for the DB?